### PR TITLE
Add Intrusion System configuration properties

### PR DESCRIPTION
**Pull request by adgrieve**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by adgrieve** [comment]
@shanyp

**Review by shanyp — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json [hunk 1]
```diff
@@ -1152,15 +1152,62 @@
       },
       "description": "Response for ListFirewallPolicyRuleGroups API service call."
     },
-    "FirewallPolicyIntrusionSystemMode": {
+    "FirewallPolicyIntrusionSystem": {
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json:1155]: Please add description for this definition

**Review by anton-evseev — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json [hunk 2]
```diff
@@ -623,9 +623,9 @@
           "description": "ThreatIntel Whitelist for Firewall Policy.",
           "$ref": "#/definitions/FirewallPolicyThreatIntelWhitelist"
         },
-        "intrusionSystemMode": {
-          "description": "The operation mode for Intrusion system.",
-          "$ref": "#/definitions/FirewallPolicyIntrusionSystemMode"
+        "intrusionSystem": {
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json:626]: Any example change needed?

**Comment by number213** [comment]
`network-may-release` has been merged into master. If this is an urgent fix please retarget to master and work with ARM to get it merged before SDK release tomorrow. Else please wait for the next release cycle.

File: specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json (hunk 2 above)
Comment (adgrieve) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json:626]: we still didn't add examples - this feature is currently available only by feature flag

**Comment by number213** [comment]
@adgrieve please re-do this change in `network-june-release` branch

**Pull request closed without merging**